Automattic/pocket-casts-android
# Improved full screen player Talkback

## Description

After reading an [issue on the player accessibility](https://github.com/Automattic/pocket-casts-android/issues/2264), I checked our TalkBack. I found some issues that I have fixed in this PR.

The following items were missing a label
- The Up Next button
- The time remaining label
- The rearrange button

## Testing Instructions
1. Enable Talkback
2. Open the full screen player
3. Navigate through the items above

✅ Verify the labels are now spoken out

## Screenshots

| Before | After |
| --- | --- |
| <img width="665" alt="Screenshot 2024-08-28 at 10 07 18 AM" src="https://github.com/user-attachments/assets/f646de7d-acde-4365-8f98-1408c49d2358"> | <img width="652" alt="Screenshot 2024-08-28 at 10 20 54 AM" src="https://github.com/user-attachments/assets/da897a5b-20b5-4a45-b9d5-f3b6809b8040"> | 
| <img width="546" alt="Screenshot 2024-08-28 at 10 07 02 AM" src="https://github.com/user-attachments/assets/0daea999-d657-420d-a482-ba76d910f978"> | <img width="527" alt="Screenshot 2024-08-28 at 10 21 07 AM" src="https://github.com/user-attachments/assets/dee748fb-9c59-4ccb-be1e-8fe36aa49ef5"> | 
| <img width="537" alt="Screenshot 2024-08-28 at 10 08 03 AM" src="https://github.com/user-attachments/assets/05f7f2f4-9ba6-4a20-ae2c-1df83554aef7"> | <img width="533" alt="Screenshot 2024-08-28 at 10 21 23 AM" src="https://github.com/user-attachments/assets/e13d6b29-fe7b-4bbb-9f3f-72ac71954768"> | 

## Checklist
